api/v1/db/oracledb/reviews-dao.js

```diff
@@ -4,18 +4,24 @@ const _ = require('lodash');
 const { serializeReview, serializeReviews } = require('../../serializers/reviews-serializer');
 
 const conn = appRoot.require('api/v1/db/oracledb/connection');
+const { openapi } = appRoot.require('utils/load-openapi');
 
 /**
  * @summary Return a list of reviews
  * @function
  * @returns {Promise<Object[]>} Promise object represents a list of reviews
  */
 const getReviews = async (queries) => {
-  const sqlParams = {};
+  // get parameters accepted by this endpoint in openapi
+  // filter params that should not be included in query
+  // gameIds is special since the values are parsed directly into the query string
+  const paramsToFilter = ['page[size]', 'page[number]', 'gameIds'];
+  const acceptedParams = openapi.paths['/reviews'].get.parameters.map(x => x.name).filter(param => !paramsToFilter.includes(param));
 
-  // get parameters from query
-  _.forEach(Object.keys(queries), (param) => {
-    if (param !== 'gameIds' && param !== 'page[size]' && param !== 'page[number]') {
+  // pick parameters specified in openapi (getReviewsParameters) from passed in queries list
+  const sqlParams = {};
+  _.forEach(acceptedParams, (param) => {
```

Comment:
This for loop can be replaced by 
`const sqlParams = _.pick(queries, acceptedParams);`

Reply:
I spent way to much time messing with pick and omit for it to be that easy.